scipy/sparse/tests/test_base.py

```diff
@@ -3461,6 +3461,167 @@ def test_argmax(self):
             assert_raises(ValueError, mat.argmax, axis=axis)
             assert_raises(ValueError, mat.argmin, axis=axis)
 
+    def test_minmax_explicit_axis(self):
```

Comment:
I think these tests would be easier to understand if they were interleaved in the existing tests. That is, update the `test_argmax` method to add more `assert_equal()` calls for the `explicit=True` case instead of adding a new `test_argmax_explicit` method.

Reply:
Done